Paramedic Lockers, Portafibs, and Chemvend Tweaks

rosieposieeee opened the pull request:
<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- The text between the arrows are comments - they will not be visible on your PR. -->

## About the PR
<!-- What did you change in this PR? -->

Added the content from my previous PRs on the original branch. As a recap of these PRs:
- [Portafibs](https://github.com/DeltaV-Station/Delta-v/pull/151) are smaller, portable defibs with less charge, for Paramedics to carry around to help people who JUST died. They hardly have enough charge to really bring back anyone who has been dead for a while, making them situational.
- [Paramedic access](https://github.com/DeltaV-Station/Delta-v/pull/197) was actually added, which means the paramedic secure windoor now works. Oh, and doctors can't nab the really valuable Paramedic items like the hud and suit and make paramedics into shittier doctors with no chem access.
- Added more of each jug to the [chemvend](https://github.com/DeltaV-Station/Delta-v/pull/199) so that restocks will be happening less constantly.

On top of that, I made additional tweaks and fixes to those previous PRs.

- Portafibs were only in duffels by accident, fixed that
- added the emagged chem dispenser chemicals to the chemvend since the dispenser will not be around very often
- gave paramed access for doctors when there's a skeleton crew
- I also made the rollerbed inside paramedic gear the emergency one since it seems more fitting, stands out more, is less ugly, and is otherwise literally identical.

https://user-images.githubusercontent.com/52761126/263178098-72f4e837-b8f3-48c7-b287-cfaf527f6178.mp4

![image](https://github.com/DeltaV-Station/Delta-v-rebase/assets/52761126/8a11dfde-8266-4c9b-a4c7-be5777e464e9)

![image](https://github.com/DeltaV-Station/Delta-v-rebase/assets/52761126/faa0bd89-b603-4c13-b70b-4868f7470db3)



Tested everything again and it all works.

## Technical details
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->
I had to add napalm and ultravasculine bottles to our own code.

## Media
<!-- 
PRs which make ingame changes (adding clothing, items, new features, etc) are required to have media attached that showcase the changes.
Small fixes/refactors are exempt.
Any media may be used in SS14 progress reports, with clear credit given.

If you're unsure whether your PR will require media, ask a maintainer.

Check the box below to confirm that you have in fact seen this (put an X in the brackets, like [X]):
-->

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
<!--
List any breaking changes, including namespace, public class/method/field changes, prototype renames; and provide instructions for fixing them. This will be pasted in #codebase-changes.
-->

**Changelog**
<!--
Make players aware of new features and changes that could affect how they play the game by adding a Changelog entry. Please read the Changelog guidelines located at: https://docs.spacestation14.io/en/getting-started/pr-guideline#changelog
-->

<!--
Make sure to take this Changelog template out of the comment block in order for it to show up.
:cl:
- add: Added the Portafib, more stuff to Chemvends, and Paramedic-only access.
-->

DebugOk commented:
You don't need to close prs to make changes, just push changes to your branch and they'll appear on the pr automatically

ShatteredSwords commented:
\o/ its passed checks let's goooo

rosieposieeee committed:
add paramedic-only access

rosieposieeee committed:
edit chemvend

upped jug amounts in each vend to accomodate for their use, and adds all emagged chemicals from the dispenser to them

rosieposieeee committed:
add portafib

FluffiestFloof reviewed (commented):
I love comments.
<img src="http://puu.sh/JR1mP/f20eb2e06e.png" data-canonical-src="http://puu.sh/JR1mP/f20eb2e06e.png" width="50" height="50" />
They are helpful in figuring what is from especially when touching the base game files

rosieposieeee committed:
added comments and tweaked rollerbed

rosieposieeee committed:
one line comments

DebugOk reviewed (changes requested):
Paramedic is not defined within [Content.Shared/Access/Components/IdCardConsoleComponent.cs](https://github.com/DeltaV-Station/Delta-v-rebase/blob/master/Content.Shared/Access/Components/IdCardConsoleComponent.cs#L45) meaning its an unobtainable access level outside of spawning with it.

rosieposieeee committed:
Update IdCardConsoleComponent.cs

rosieposieeee committed:
Merge remote-tracking branch 'upstream/master' into medical

DebugOk reviewed (changes requested):
Test failures are valid. Chemvend changes introduce free money for cargo.

```A total of 1 test files matched the specified pattern.
  Failed NoCargoOrderArbitrage [1 s]
  Error Message:
     Found arbitrage on CrateVendingMachineRestockChemVend cargo product! Cost is 3820 but sell is 6770!
  Expected: less than or equal to 3820
  But was:  6770.0d```

rosieposieeee committed:
increase crate price

Colin-Tel committed:
Comment, misspelling adjustments

removed unnecessary comments (ones indicating a Delta V change while already in the Delta V directory)

Colin-Tel reviewed (dismissed):
Looks good! We'll just have to see how this affects the balance of things.

Colin-Tel committed:
Merge branch 'master' into medical

Signed-off-by: Colin-Tel <113523727+Colin-Tel@users.noreply.github.com>

Colin-Tel reviewed (dismissed):
*me saying the same thing post-merge fix*
Looks good! We'll just have to see how this affects the balance of things.

rosieposieeee committed:
not all jugs are created equal

this wont work btw

rosieposieeee committed:
chemvend restock price
